src/components/List/List.tsx

```diff
@@ -131,20 +149,29 @@ class List extends UIComponent<Extendable<ListProps>, ListState> {
 
   renderItems() {
     const { items } = this.props
-    const { selectedItemIndex } = this.state
+    const { focusedIndex, selectedIndex } = this.state
 
     this.itemRefs = []
 
     return _.map(items, (item, idx) => {
       const maybeSelectableItemProps = {} as any
 
-      if (this.props.selection) {
+      if (this.props.selectable) {
         const ref = React.createRef()
         this.itemRefs[idx] = ref
 
-        maybeSelectableItemProps.tabIndex = idx === selectedItemIndex ? 0 : -1
         maybeSelectableItemProps.ref = ref
-        maybeSelectableItemProps.onFocus = () => this.focusHandler.syncFocusedItemIndex(idx)
+        maybeSelectableItemProps.onFocus = () => this.focusHandler.syncFocusedIndex(idx)
+        maybeSelectableItemProps.onClick = e => {
+          this.trySetState({ selectedIndex: idx })
+          _.invoke(this.props, 'onSelectedIndexChange', e, {
+            ...this.props,
+            ...{ selectedIndex: idx },
+          })
+          this.setState({ focusedIndex: idx })
```

Comment:
here we should call `this.focusHandler.syncFocusedIndex(idx)` - this 'setState()' logic will be part of it. Otherwise the problem is that we won't sync focused state with the focusManager